Flow.Launcher/MainWindow.xaml

```diff
@@ -83,24 +84,29 @@
                      AllowDrop="True"
                      Visibility="Visible"
                     Background="Transparent"
-                    Margin="18,0,56,0">
+                    Margin="16,0,56,0">
                         <TextBox.ContextMenu>
                             <ContextMenu>
                                 <MenuItem Command="ApplicationCommands.Cut"/>
                                 <MenuItem Command="ApplicationCommands.Copy"/>
                                 <MenuItem Command="ApplicationCommands.Paste"/>
                                 <Separator />
-                                <MenuItem Header="Settings" Click="OnContextMenusForSettingsClick" />
+                                <MenuItem Header="{DynamicResource flowlauncher_settings}"  Click="OnContextMenusForSettingsClick" />
                             </ContextMenu>
                         </TextBox.ContextMenu>
                     </TextBox>
-                    <svgc:SvgControl Source="{Binding Image}" HorizontalAlignment="Right" Width="42" Height="42"
-                                     Background="Transparent"/>
+                    <svgc:SvgControl Source="{Binding Image}" HorizontalAlignment="Right" Width="25" Height="25" Margin="0,2,18,0" Opacity="0.2" Background="Transparent" Style="{DynamicResource SearchIconStyle}"/>
                 </Grid>
-                <Line x:Name="ProgressBar" HorizontalAlignment="Right"
+
+
+                <Grid>
+                    <Rectangle Width="Auto" HorizontalAlignment="Stretch" Style="{DynamicResource SeparatorStyle}" />
+                    <Line x:Name="ProgressBar" HorizontalAlignment="Right"
                   Style="{DynamicResource PendingLineStyle}" Visibility="{Binding ProgressBarVisibility, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" 
-                  Y1="0" Y2="0" X2="100" Height="2" Width="752" StrokeThickness="1">
-                </Line>
+                  Y1="0" Y2="0" X2="100" Height="2" Width="Auto" StrokeThickness="1">
```

Comment:
I haven't tested it, but progressbar will come out nicely as it is placed on the separator. you guys talking about using spinner, I think this is better.

Reply:
I tried to get the progress bar shown but either dont have too many files or the progress bar is not showing up, could both of you test when you get a chance please @taooceros